lib/pharos/phases/configure_master.rb

```diff
@@ -296,6 +298,25 @@ def generate_authentication_token_webhook_config(webhook_config)
         config
       end
 
+      # @param config [Hash]
+      def configure_kube_proxy(config)
+        config['kubeProxy'] = {
+          'config' => {
+            'featureGates' => {}
+          }
+        }
+
+        if @config.kube_proxy.mode
+          config['kubeProxy']['config']['mode'] = @config.kube_proxy.mode
+        end
+
+        if @config.kube_proxy.mode == 'ipvs'
+          config['kubeProxy']['config']['featureGates']['SupportIPVSProxyMode'] = true
+        end
+
+        config
```

Comment:
This return is only here to workaround bad rubocop suggestions:

```
lib/pharos/phases/configure_master.rb:313:9: C: Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.
        if @config.kube_proxy.mode == 'ipvs'
        ^^
```